src/patterns.rs

```diff
@@ -133,7 +133,7 @@ impl Rewrite for Pat {
                     .iter()
                     .map(|p| {
                         if let Some(rw) = p.rewrite(context, shape) {
-                            format!("{}", if rw == "_" { "" } else { &rw })
+                            format!("{}", rw)
```

Comment:
We don't have to use `format!` here:
```suggestion
                            rw
```

Reply:
oh! I fixed it.